app.coffee

```diff
@@ -86,7 +86,8 @@ app.post "/webhook", (req, res) ->
   adminNick = req.body.user_name
 
   # If the message is not meant to be sent to jinora users, but it is a command meant to be interpreted by jinora
-  isCommand = (req.body.text[0] == "!")
+  isCommand = ((req.body.text[0] == "!") 
+                || ((req.body.text[0] == "/") && (req.body.text[0] == "/")))
```

Comment:
Shouldn't it be `req.body.text[1]` in the third place?
Also a more better way would be to use [startsWith()](https://www.w3schools.com/jsref/jsref_startswith.asp) method.